tests/mbt/model/ccv_pss_test.qnt

```diff
@@ -0,0 +1,34 @@
+// This module contains logic for PSS (Partial Set Security).
+// PSS is a variant/extension of CCV that 
+// allows for only a subset of the validator set
+// to secure a consumer chain.
+// Not all logic related to PSS is inside this module, as some logic is
+// too tightly coupled with the core CCV logic,
+// which is instead found in ccv.qnt
+module ccv_pss_test {
+    import ccv_types.* from "./ccv"
+    import extraSpells.* from "./libraries/extraSpells"
+    import ccv_utils.* from "./ccv_utils"
+    import ccv_pss.* from "./ccv_pss"
+
+    run TopNTest = 
+        val valSet =
+            Map("a" -> 5, "a1" -> 5, "a2" -> 5, "b" -> 10, "b1" -> 10, "b2" -> 10, "c" -> 15, "c1" -> 15, "e" -> 25)
+        // total power: 5*3 + 10*3 + 15*2 + 25 = 100
+        all
+        {
+            assert(GetTopNVals(valSet, 0) == Set()),
+            assert(GetTopNVals(valSet, 1) == Set("e")),
+            assert(GetTopNVals(valSet, 10) == Set("e")),
+            assert(GetTopNVals(valSet, 25) == Set("e")),
+            // if one validator with a power is included, all validators with that power need to be included
+            assert(GetTopNVals(valSet, 26) == Set("e", "c1", "c")),
+            assert(GetTopNVals(valSet, 45) == Set("e", "c1", "c")),
+            assert(GetTopNVals(valSet, 55) == Set("e", "c1", "c")),
+            assert(GetTopNVals(valSet, 56) == Set("e", "c1", "c", "b2", "b1", "b")),
+            assert(GetTopNVals(valSet, 85) == Set("e", "c1", "c", "b2", "b1", "b")),
+            assert(GetTopNVals(valSet, 86) == Set("e", "c1", "c", "b2", "b1", "b", "a2", "a1", "a")),
+            assert(GetTopNVals(valSet, 95) == Set("e", "c1", "c", "b2", "b1", "b", "a2", "a1", "a")),
+            assert(GetTopNVals(valSet, 100) == Set("e", "c1", "c", "b2", "b1", "b", "a2", "a1", "a")),
```

Comment:
nit: Why not `GetTopNVals(valSet, 100) == valSet`?

Reply:
Sure. need to be valSet.keys(), because valSets are actually maps.. but still more descriptive :)